crossfire/__init__.py

```diff
@@ -51,16 +59,44 @@ def occurrences(
     )
 
 
+def _flatten_df(row, column_name):
+    column_data = row[column_name]
+    return Series(
+        {f"{column_name}_{key}": value for key, value in column_data.items()}
+    )
+
+
+def is_empty(data):
+    if not HAS_PANDAS or (HAS_PANDAS and not isinstance(data, DataFrame)):
+        return not data
+    elif isinstance(data, DataFrame):
+        return data.empty
+    else:
+        return False
```

Comment:
What is the use case for this `else` branch?

Overall I think the readability is more complex than it could be. In my mind what we need is two main branches of logic:
* If `HAS_PANDAS` we check if data is a `DataFrame` and, if so, return `data.empty`
* If not, just return the boolean representation of `data`

Is there any use case I am leaving behind?

Reply:
It make sense. I don't now how. But yesterday the way I wrote was making sense...
Mainly the first branch, checking if the user hasn't Pandas...
Any way, I ahve updated using your suggestion.